django/db/models/fields/tuple_lookups.py

```diff
@@ -271,6 +304,13 @@ def as_sqlite(self, compiler, connection):
 
         return root.as_sql(compiler, connection)
 
+    def as_subquery(self, compiler, connection):
+        if not connection.features.supports_tuple_in_subquery:
+            raise NotSupportedError(
+                "TupleIn subqueries are not supported on this database backend."
```

Comment:
Can we mention which database backend that is? It's possible to configure Django to use more than one, in which case maybe the programmer meant to use a different one.